cxotime/__init__.py

```diff
@@ -5,6 +5,7 @@
 
 from .convert import *  # noqa: F401, F403
 from .cxotime import *  # noqa: F401, F403
+from .utils import get_range_in_chunks
```

Comment:
Generally you should follow the same pattern `from .utils import *` in the top-level `__init__` and then define the imports with `__all__`. But I'm recommending to do away with `utils` so this would be OBE.